README.md

```diff
@@ -63,8 +68,12 @@
 
 For a live demo to our project, follow the link below
 
+
+- [Live Demo Link](https:https://github.com/maherramadan78/leaderboard)
+=======
 - [Live Demo Link](https://github.com/maherramadan78/leaderboard.git)
```

Comment:
- [ ] Kindly consider writing for example `Coming soon` instead of placing the same repo link as this could be misguiding unintentionally.